### PR TITLE
fix testing for editor

### PR DESCRIPTION
**Pull request by chrismgeorge**
- added karma-spec-reporter for visualizing the test cases
- fixed broken model_viewer_preview_test
- commented out expandable_contents test for content (because of change to UI) - will update later if really needed

### CODE REVIEW

File: packages/space-opera/karma.conf.js [hunk 1]
```diff
@@ -22,22 +22,32 @@
 module.exports = function(config) {
   config.set({
     basePath: '',
+    reporters: ['spec'],
+    specReporter: {
+      maxLogLines: 10,              // limit number of lines logged per test
+      suppressErrorSummary: false,  // do not print error summary
```
Comment (elalish) [inline packages/space-opera/karma.conf.js:28]: Are these comments correct? It seems like setting suppress to false would print an error summary.

File: packages/space-opera/karma.conf.js [hunk 2]
```diff
@@ -22,22 +22,32 @@
...
     plugins: [
       require.resolve('@open-wc/karma-esm'),
       'karma-*',
+      'karma-spec-reporter'
     ],
     frameworks: ['esm', 'jasmine'],
     files: [
       './define_process_env.js',
-      {pattern: 'lib/**/*_test.js', watched: true, type: 'module'},
+      {pattern: 'lib/**/*_test.js', watched: false, type: 'module'},
```
Comment (elalish) [inline packages/space-opera/karma.conf.js:42]: Was this intentional?

File: packages/space-opera/karma.conf.js [hunk 3]
```diff
@@ -22,22 +22,32 @@
...
     ],
     autoWatchBatchDelay: 1000,
     restartOnFileChange: true,
 
     browserDisconnectTimeout: 300000,
     browserNoActivityTimeout: 360000,
     captureTimeout: 420000,
-    concurrency: 10,
+    concurrency: 1,
```
Comment (elalish) [inline packages/space-opera/karma.conf.js:50]: Does this help with keeping the testing order more clear?

File: packages/space-opera/src/components/shared/expandable_content/expandable_content_test.ts [hunk 4]
```diff
@@ -66,7 +66,7 @@ describe('<me-expandable-section>', () => {
     section.open = true;
     await section.updateComplete;
 
-    expect(elementIsVisible(section)).toBe(true);
+    // expect(elementIsVisible(section)).toBe(true);
```
Comment (elalish) [inline packages/space-opera/src/components/shared/expandable_content/expandable_content_test.ts:69]: Maybe put a TODO so it's easier to find in the future?

**Review by elalish — DISMISSED** [review]

File: packages/space-opera/karma.conf.js (hunk 1 above)
Comment (chrismgeorge) [reply packages/space-opera/karma.conf.js:28]: whoops, lemme clean that up

File: packages/space-opera/karma.conf.js (hunk 2 above)
Comment (chrismgeorge) [reply packages/space-opera/karma.conf.js:42]: no, I'll revert

File: packages/space-opera/karma.conf.js (hunk 3 above)
Comment (chrismgeorge) [reply packages/space-opera/karma.conf.js:50]: it doesn't seem to have any significant effect, so I limited it

File: packages/space-opera/src/components/shared/expandable_content/expandable_content_test.ts (hunk 4 above)
Comment (chrismgeorge) [reply packages/space-opera/src/components/shared/expandable_content/expandable_content_test.ts:69]: gotcha

**Review by elalish — APPROVED** [review]

**Review by elalish — APPROVED** [review]

**Pull request merged**